jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -479,3 +479,29 @@
         export TEST_CLUSTER_DELETE_COLLECTION_WORKERS="--delete-collection-workers=16"
     jobs:
         - 'kubernetes-e2e-{suffix}'
+
+# Jobs that run e2e tests on GCE with container-vm-v20160321 image to do some
+# experimental (docker 1.9.1 and Kubernetes 1.3) for
+# https://github.com/kubernetes/kubernetes/issues/27691
+- project:
+    name: kubernetes-e2e-gce-pd
+    trigger-job: 'kubernetes-build'
+    test-owner: 'saadali@google.com'
+    provider-env: ''
+    emails: 'saadali@google.com,dawnchen@google.com'
+    suffix:
+        - 'gce-pd-test':
+            description: 'Run E2E slow tests on GCE test endpoint.'
+            timeout: 150
+            job-env: |
+                export GINKGO_TEST_ARGS="--ginkgo.focus=RW\sPD,\sremove\sit,\sthen\sschedule\sit\son\sanother\shost \
```

Comment:
Oh, nevermind. Bug in how I was testing. However, please take out the `--ginkgo.skip` section below.